syntax = "proto2";

message PandaLensReset {
  optional string message = 1 [default = ""];
}